File: class Solution{      static int inSequence(int A, int B, int C){/code here          if(C ==0 ) {              if(A != B) return 0;              return 1;          }          int n = (B-A)/Arithmetic Number.java
class Solution{

    static int inSequence(int A, int B, int C){

        // code here

        if(C ==0 ) {

            if(A != B) return 0;

            return 1;

        }

        int n = (B-A)/C; 

        if(n < 0) return 0;

        int rem= (B-A)%C; 

        if(rem == 0 ) return 1;

        else return 0;

    }

}

File: Arithmetic Number.cpp
class Solution{

public:

    int inSequence(int A, int B, int C)

    {

        if(A==B)

        {

            return 1;

        }

        

        if(B!=A && C==0)

        {

            return 0;

        }

        

        int num=B-A;

        if(num<0)

        {

            if(C<0 && num%C==0)

            {

                return 1;

            }

        }

        else

        {

            if(C>0 && num%C==0)

            {

                return 1;

            }

        }

        return 0;

    }

};

